brave/browser-laptop
# Fix Sync after sites refactor

See also: #10136 (sites refactor)

Submitter Checklist:


Test Plan:
0. Prepare a user profile with 1k+ bookmarks for Pyramid 0.
1. Start Pyramid 0 and enable Sync.
2. Start Pyramid 1 and join the Sync profile.
3. Wait for syncing to complete. To confirm, you can check both pyramids's consoles for "got 0 bookmarks".
4. Export bookmarks from both pyramids and compare with https://github.com/brave/browser-laptop/pull/10166 – they should be identical.

Reviewer Checklist:

Tests
